resources/content/schema/base/container.json

```diff
@@ -235,6 +235,14 @@
          "startCount" : {
             "type" : "int",
             "nullable" : true
+        },
+        "blkioDeviceReadIOps": {
+            "type": "array[map[string]]",
```

Comment:
they type of the array shouldnt be a map. it should be an explicit object of type iops. 

You can follow the haConfig option as an example on how to do this. You need:
- an iops.josn file in the same directory as the haConfig.json file
- an entry in spring-core-model-context.xml, same place where haConfig is defined
- add permissions in user-auth.json
- change the type from map to this concrete type